### PR TITLE
BHBC-1965

### PR DESCRIPTION
**Pull request by cgarrettjones**
# Overview

## Links to Jira tickets

- [BHBC-1965](https://quartech.atlassian.net/browse/BHBC-1965)
- [BHBC-1967](https://quartech.atlassian.net/browse/BHBC-1967)

## Description of relevant changes

 - addition of species descriptor to survey occurrence templates 
 - addition of templates concept to survey summary details

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

### CODE REVIEW

**Review by KjartanE — DISMISSED** [review]
Looks good! Applications functions

File: database/src/migrations/20221005063930_api_delete_survey copy.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { Knex } from 'knex';
```
Comment (NickPhura) [inline database/src/migrations/20221005063930_api_delete_survey copy.ts]: The name of this file is a bit messed up, has `copy` at the end.

**Review by NickPhura — CHANGES_REQUESTED** [review]
Just a small fix needed for the file name of the one migration.

File: database/src/migrations/20221005063930_api_delete_survey copy.ts (hunk 1 above)
Comment (cgarrettjones) [reply database/src/migrations/20221005063930_api_delete_survey copy.ts]: please find adjusted.

**Review by NickPhura — APPROVED** [review]

**Comment by NickPhura** [comment]
Looks like the setup is failing because the migration table is 'corrupt'.  Its probably due to that one migration having its file name changed, after having run under the old name.

https://console.apps.silver.devops.gov.bc.ca/k8s/ns/af2668-dev/pods/biohubbc-db-setup-dev-830/logs

Solution is to just blow away the PR artifacts in OpenShift, and then it can re-build from scratch.

**Pull request merged**